testing/internal-testing/src/main/groovy/org/gradle/integtests/fixtures/SkippedExecutionResult.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures;
+
+public class SkippedExecutionResult {
+    private final String message;
+    private final String type;
+    private final String text;
+
+    public SkippedExecutionResult() {
```

Comment:
Nitpick: The use case for this constructor seems like an edge case rather than something common enough to define a dedicated constructor for. Can we just use the regular constructor for the one time we call this one?